recipes/z5py/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.5.3" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+package:
+  name: z5py
+  version: {{ version }}
+
+source:
+    url: https://github.com/constantinpape/z5/archive/{{ version }}.tar.gz
+    # sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - boost-cpp
+    - numpy
+    - xtensor >=0.17.1,<0.18
+    - xtensor-python
+    - bzip2
+    - c-blosc
+    - xz
+    - zlib
```

Comment:
I should have added that comment earlier: in general I would recommend putting version ranges for libraries that are not pinned and are likely to change rapidly. Maybe `xtensor-python >0.20,<0.21` would also hepl.

Reply:
Thanks for the clarification.

Reply:
If it helps, we can add pinnings to [the global pinning file]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml ).

Reply:
We have been updating `xtensor-python` quite often, so I am not sure this is a good idea to globally pin it.